playbooks/ceph_client.yml

```diff
@@ -1,5 +1,13 @@
 ---
-
+- name: Check remote connections
+  hosts: "{{ edpm_override_hosts | default('all', true) }}"
+  strategy: linear
+  gather_facts: "{{ gather_facts | default(false) }}"
+  tasks:
+    - name: Wait for connection
+      ansible.builtin.wait_for_connection:
+        delay: "{{ edpm_wait_for_connection_delay | default(10) }}"
+        timeout: "{{ edpm_wait_for_connection_timeout | default(600) }}"
```

Comment:
can we put this task into a common role? I'm thinking edpm_nodes_validation, in a separate tasks file that can be included.

Reply:
yes, I think that makes sense